apps/chat/src/components/Common/MultipleComboBox.tsx

```diff
@@ -240,7 +245,11 @@ export function MultipleComboBox<T>({
                   contentClassName="text-xs"
                 >
                   <span
-                    className="flex h-[23px] items-center justify-between gap-2 rounded bg-accent-primary-alpha px-2 py-1.5"
+                    className={classNames(
+                      'flex items-center justify-between gap-2 rounded bg-accent-primary-alpha px-2 py-1.5',
+                      { 'h-[23px]': !itemHeight },
+                      { [`h-[${itemHeight}]`]: itemHeight },
```

Comment:
as far as I know this code will not work. If you need to change styles dynamically, you need to move this line to `styles` prop